dcrsqlite/apisource.go

```diff
@@ -320,12 +317,83 @@ func (db *wiredDB) GetRawTransactionWithPrevOutAddresses(txid string) (*apitypes
 
 	msgTx := txhelpers.MsgTxFromHex(txhex)
 	if msgTx == nil {
-		return tx, prevOutAddresses, "", nil
+		return &apitypes.ExplorerTxData{
+			Tx:       tx,
+			VinAddrs: prevOutAddresses,
+		}
 	}
 
 	txType := txhelpers.DetermineTxTypeString(msgTx)
+	var txFee, txFeeRate float64
+	if txType == "Ticket" || (txType == "Regular" && len(tx.Vin[0].Coinbase) < 5) {
```

Comment:
Revocations too have a fee.  You might be better off reversing this and checking for votes and regular coinbase.

But also i don't follow the `len(tx.Vin[0].Coinbase) < 5` part.  What is 5?

Reply:
Used it because when there is no coinbase the coinbase string consists of 4 spaces

Reply:
oh, that's interesting.  Is that set somewhere explicitly in dcrd?  we should check that out

Reply:
I don't know if it's set somewhere in dcrd, just an observation I made while debugging